gridlab-d/gridlab-d
# Merge #1260 - deltamode "module order" feature

#### What's this Pull Request do?
Merges the changes in #1260, which implements the deltamode "module execution order" guidance.  Makes it so deltamode sequences things properly, regardless of the GLM order of the modules.

#### Where should the reviewer start?
Pull, build, and run the autotests.

#### How should this be tested?
Autotests.  To see how order may affect things, powerflow/autotest/test_multi_island_SWING_PQ_Indep_Islands_sync_check.glm was adjusted to use the functionality.  Add in a `#set deltamode_preferred_module_order=false` line to see how it would fail without the fixes in #1260.

#### What are the relevant issues?
#1260
